tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/ui/models/ResourceEquality.java

```diff
@@ -0,0 +1,27 @@
+package org.jboss.tools.openshift.test.ui.models;
+
+import com.openshift.restclient.model.IResource;
+
+public class ResourceEquality {

```

Comment:
I just use it here to enable my handmade mocks. Unfortunately, Mockito does not allow for equals() to be mocked.